src/components/App/ActiveShowModal/index.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { hideDetails } from 'redux/actions/detailsActions';
+
+import imgMissing from 'images/img_missing.png';
+
+import styles from './ActiveShowModal.module.scss';
+
+class ActiveShowModal extends Component {
+  
+  render() {
+    const { isVisible, activeShow } = this.props;
+    return (
+      <div className={`${isVisible ? styles.visible : ''} ${styles.modalWrapper}`}>
+        <div className={styles.modalContent}>
+          <button
+            className={styles.close}
+            onClick={() => this.props.hideDetails()}
+          >
+            &times;
+          </button>
+          <figure>
+            <img
+              src={activeShow && activeShow.activeShow.show.image.medium ? activeShow.activeShow.show.image.medium : imgMissing}
+              alt="" />
+            <figcaption>
+              <h5 className={styles.showTitle}>
+                {activeShow ? activeShow.activeShow.name : ''}
```

Comment:
If there is a lot of conditions for `activeShow` maybe consider another function with jsx for activeShow and render it once conditionally?